cmd/node/config/genesis.json

```diff
@@ -1,122 +1,1322 @@
 [
```

Comment:
Do we need to change the default values in this file ?

Reply:
yes, the node, at the current moment can not be started directly (to check the binary works) due to the fact that there is a mismatch between the staked value per node in genesis files (500k) and the economics.toml (2.5 mil)